ReSharper.FSharp/src/FSharp.Psi.Features/src/QuickFixes/ReplaceLambdaWithInnerExpressionFix.fs

```diff
@@ -12,6 +13,10 @@ type ReplaceLambdaWithInnerExpressionFix(warning: LambdaCanBeReplacedWithInnerEx
     override x.Text =
         match replaceCandidate with
         | :? IReferenceExpr as ref when isSimpleQualifiedName ref ->
-            sprintf "Replace lambda with '%s'" ref.QualifiedName
-        | :? IPrefixAppExpr -> "Replace lambda with partial application"
+            $"Replace lambda with '{ref.QualifiedName}'"
+        | :? IPrefixAppExpr as app ->
+            match getRootFunctionExpr app with
+            | :? IReferenceExpr as ref when isSimpleQualifiedName ref ->
+                $"Replace with '{ref.QualifiedName}' partial application"
```

Comment:
Is there a reason why a qualified name makes it better, even though the text may get considerably longer? Does it also properly work with all kind of arbitrary qualifier expressions?

Reply:
No reply, but it's resolved.